decidim-meetings/spec/system/admin/filter_meetings_spec.rb

```diff
@@ -11,7 +11,7 @@
 
   include_context "when managing a component as an admin"
 
-  TYPES = Decidim::Meetings::Meeting::TYPE_OF_MEETING.map(&:to_sym)
+  TYPES = Decidim::Meetings::Meeting::TYPE_OF_MEETING.keys.map(&:to_sym)
```

Comment:
Isn't it already a symbol? I think we can remove `.to_sym`.
```suggestion
  TYPES = Decidim::Meetings::Meeting::TYPE_OF_MEETING.keys
```